ckanext/switzerland/dcat/profiles.py

```diff
@@ -46,12 +48,69 @@
     'xml': XML,
 }
 
-ogd_theme_base_url = 'http://opendata.swiss/themes/'
+ogd_theme_base_url = 'http://opendata.swiss/themes'
 
 slug_id_pattern = re.compile('[^/]+(?=/$|$)')
 
 
-class SwissDCATAPProfile(RDFProfile):
+class MultiLangProfile(RDFProfile):
+    def _add_multilang_value(self, subject, predicate, dataset_key=None, dataset_dict=None, multilang_values=None):  # noqa
+        if not multilang_values and dataset_dict and dataset_key:
+            multilang_values = dataset_dict.get(dataset_key)
+        if multilang_values:
```

Comment:
not sure if it makes sense to provide a fallback and just add the subject and predicate to the graph as a simple Literal if that is not misleading?

Reply:
Pretty much, what is being done in https://github.com/opendata-swiss/ckanext-switzerland/pull/177/files#diff-3ec9836d38fb4719275ebd0bd7cd202bR74